ceremonies/src/lib.rs

```diff
@@ -1156,7 +1168,7 @@ pub mod pallet {
 		CommunityCeremony,
 		Blake2_128Concat,
 		MeetupIndexType,
-		MeetupResult,
+		Option<MeetupResult>,
 		ValueQuery,
```

Comment:
Instead of making this an `Option<MeetupResult>`, you should change `ValueQuery` to `OptionQuery`.